### ISSUE TITLE
Getting intermittent 'Already Seen' messages on the P4: T goldstandard workflow and also getting same images I've already classified without the aleready seen flag

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes#1932]
I'm classifying gold standard data for Planet Four: Terrains (workflow 2063). The idea was that two science team members would classify a small number of subjects on the inactive workflow by going directly to the url. I'm going to use the gold standard data for my own checks (it's not going into the main workflow for feedback). 

I'm starting to get intermediate 'Already Seen' tags on images and then the next  image doesn't have that and then if I reload or log back in I can see another image which is actually a subject I know I classified recently. 

I'm confused as this is a small number of subjects and only 2 people classifying why panoptes would be having issues with the already seen flag and what to send to me to classify for the workflow. I thought I'd put in a ticket.

<img width="1123" alt="screen shot 2016-08-31 at 3 13 51 pm" src="https://cloud.githubusercontent.com/assets/753325/18152263/46d12db4-6f90-11e6-8395-2dfb96516303.png">

<img width="1113" alt="screen shot 2016-08-31 at 3 13 54 pm" src="https://cloud.githubusercontent.com/assets/753325/18152257/42bd9c4e-6f90-11e6-8bd6-6015fe0bad83.png">

then go this image again without the tag
<img width="1121" alt="screen shot 2016-08-31 at 3 27 45 pm" src="https://cloud.githubusercontent.com/assets/753325/18152267/4b69c37c-6f90-11e6-8b77-da05399e74ee.png">

**Comment by camallen** [comment]
this is an issue with some newly deployed code blocking the background workers, we're looking into it.

**Comment by srallen** [comment]
Snapshot WI has reported to me that they've had volunteers getting 'Already Seen' and 'Finished' banners before the subject set is done the past couple of days: https://www.zooniverse.org/projects/zooniverse/snapshot-wisconsin/talk/391/85128?comment=159761&page=1

Let me know if you want any info from them.

**Comment by camallen** [comment]
This was caused by new counters on the backend and should be fixed since 1st Sept 2016, 17:00. If this is still occuring please reopen.

**Comment by mschwamb** [comment]
@camallen I can't open since I not part of this repo only the front end. 

Looks like there's still an issue see [here](https://www.zooniverse.org/projects/mschwamb/comet-hunters/talk/210/40383?comment=160684).

**Comment by camallen** [comment]
Thanks @mschwamb it seems the some workflows were reporting as finished, notably when they had more total retired subjects than currently linked data.

**Comment by camallen** [comment]
thanks @srallen and @mschwamb this should be resolved by #1944. Please re-open / ping me if not.

**Comment by mschwamb** [comment]
@camallen this issue isn't fixed. Got this today for the other P4:T researcher:

> I think, I am done. Orange tag with "Finished" has showed up. It was a bit weird that before that happened, some frames showed "already seen" tag. I still went on and marked couple of those and the page went back to showing me new frames for a while. So, check it, if I am really through and if some answered twice.

**Comment by mschwamb** [comment]
@camallen bumping this again so it doesn't get lost.

**Comment by camallen** [comment]
@mschwamb this is most likely happening by design, perhaps they've finished the data?

**Comment by mschwamb** [comment]
What do you mean by design? It shouldn't be intermittent. If there are unseen subjects that are not retired those should go first. Getting this message already seen and then the next subject is unclassfied  means I get duplicate classifications I need to take out of my gold standard data classifications and that I have to manually check when the expert is done and tell her to stop and just keep going through the already seen to finsih the last few subjects.

**Comment by mschwamb** [comment]
I've checked. Anya's got 40 more images to go to complete the subject set, so she should be getting only new images as there is still stuff to see. It's a small set only 1000 subjects.

**Comment by camallen** [comment]
@mschwamb I've PM'd you via zooniverse.org to get more details.

**Comment by camallen** [comment]
@mschwamb that workflow is finished and all data is retired for those two subject sets. The selector in this instance will just show any data to any requesting user in this case.

**Comment by mschwamb** [comment]
Um.. no the other researcher has not finished. I'm counting everything up in the annotations csv file generated by panoptes and she has not finished all the gold standard data.  Example: for subject 484327 in the gold standard workflow the only classification in there is from me from the workflow, so how can the subject be retired if the retirement threshold s 2. The subject is retired from the main classification interface workflow but it shouldn't be in workflow 2063.  I checked again and got the annotations csv a few hours ago.  

I don't think I'm missing something. The subject csv file says its retired which I just downloaded, but where's the annotations from that user. I don't count from the subject csv, I actually calculate by adding up the classifications.

**Comment by mschwamb** [comment]
same thing for subjects:

1320489
1058430
1501258

I don't see the classifications from Anya searching these in the raw annotations file. Is there a counting issue going on?

**Comment by camallen** [comment]
@mschwamb looks like we are counting some things twice, i'm investigating.

**Comment by camallen** [comment]
@mschwamb this was a counting issue when subjects have been re-used across workflows. Once that PR is merged / deployed i'll manually update those counters for those subject sets and un-retired the data.

**Comment by camallen** [comment]
Done - @mschwamb that workflow should be showing data correctly now.
